requirements.txt

```diff
@@ -1,3 +1,4 @@
+cryptography==3.1.1
```

Comment:
Did you want to commit any changes in this file? It seems that you don't use those dependencies.

Reply:
When I try to run it locally one lib throws an error about missing Rust compiler and that cryptography cannot be built (like here: https://stackoverflow.com/questions/59441794/error-could-not-build-wheels-for-cryptography-which-use-pep-517-and-cannot-be-i)